stacks-core/create-source-binary/build-scripts/Dockerfile.linux-glibc-arm64

```
@@ -5,6 +5,7 @@ ARG GIT_BRANCH='No Branch Info'
 ARG GIT_COMMIT='No Commit Info'
 ARG BUILD_DIR=/build
 ARG TARGET=aarch64-unknown-linux-gnu
+ARG TAG="No Tag Info"
```

```suggestion
ARG TAG
```
this should be enough - specifying a string for the tag isn't necessary if it's not set (i.e. the case will check for `singer-*` only)